ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -138,7 +138,8 @@ private[state] object Conversions {
       submitter = Party.assertFromString(subInfo.getSubmitter),
       applicationId = LedgerString.assertFromString(subInfo.getApplicationId),
       commandId = LedgerString.assertFromString(subInfo.getCommandId),
-      maxRecordTime = parseTimestamp(subInfo.getMaximumRecordTime)
+      maxRecordTime = parseTimestamp(subInfo.getMaximumRecordTime),
```

Comment:
Minor: it would make sense to make the naming fully consistent here (`maxRecordTime` vs. `SubmitterInfo`'s `maximumRecordTime`).